soh/src/overlays/actors/ovl_En_Zl3/z_en_zl3.c

```diff
@@ -2580,7 +2580,7 @@ void func_80B59AD0(EnZl3* this, PlayState* play) {
     func_80088AA0(180);
     func_80B54EA4(this, play);
     func_80B53614(this, play);
-    gSaveContext.eventChkInf[12] &= ~0x80;
+    CLEAR_EVENTCHKINF(EVENTCHKINF_WATCHED_GANONS_CASTLE_COLLAPSE_CAUGHT_BY_GERUDO);
```

Comment:
just curious about the reasoning behind using a macro for clear when there are functions for the rest of the things

Reply:
Good question, clear functions dont exist yet for any of the flag types but I'll go ahead and make them and use that instead, since we'll also want the clears to be emitted in hooks

Reply:
Pushed 👍